source/FreeRTOS_TCP_Utils_IPv6.c

```diff
@@ -82,18 +82,10 @@ void prvSocketSetMSS_IPV6( FreeRTOS_Socket_t * pxSocket )
             /* Compared to IPv4, an IPv6 header is 20 bytes longer.
              * It must be subtracted from the MSS. */
             size_t uxDifference = ipSIZE_OF_IPv6_HEADER - ipSIZE_OF_IPv4_HEADER;
-            /* Do not allow MSS smaller than tcpMINIMUM_SEGMENT_LENGTH. */
```

Comment:
I think we should have this check in place, as MTU size can be updated according to user. [Example](https://www.freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/TCP_IP_Configuration_Examples.html)

Reply:
It's checked in [FreeRTOSIPConfigDefault.h](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/782/files#diff-577fb46f111095d87818a991acfb003ba427900e2bc7c915a6a856990fe7f5f7R1406-R1408). So it's not necessary to check it again here.